  INFO root [2019-09-04 20:33:04,836] - 
  INFO root [2019-09-04 20:33:04,838] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:33:04,839] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:33:04,839] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:33:04,839] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:33:04,878] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56215 )
  INFO root [2019-09-04 20:33:04,904] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56215, SOMAXCONN=5.
  INFO root [2019-09-04 20:33:04,919] - server-configs:
  INFO root [2019-09-04 20:33:04,919] - 	internalTcpAddr : 192.168.56.1:56215
  INFO root [2019-09-04 20:33:04,920] - 	componentID : 6000
  WARN root [2019-09-04 20:33:04,930] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:33:05,091] - Components::findLogger: found logger, addr:192.168.56.1:56212
  INFO root [2019-09-04 20:33:05,092] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:33:05,092] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:33:05,287] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 7792 ----
  INFO root [2019-09-04 20:33:05,287] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:33:05,288] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:33:05,288] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:33:05,795] - ---- cellappmgr is running ----
 DEBUG root [2019-09-04 20:33:05,802] - Components::process(): Request for the process of identity...
  WARN root [2019-09-04 20:33:12,284] - Forwarding logs to logger[192.168.56.1:56212/0/0/0]...
 ERROR root [2019-09-04 20:33:33,063] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=456491958054, lastReceivedTime=456429386373, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-04 20:33:33,072] - The following logs sent to logger failed:
  WARN root [2019-09-04 20:33:33,105] - ==> [2019-09-04 20:33:33 063] TCPPacketReceiver::processPendingEvents(192.168.56.1:56212): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-04 20:33:33,106] - ==> [2019-09-04 20:33:33 064] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=456491958054, lastReceivedTime=456429386373, inactivityExceptionPeriod=134887628)
  WARN root [2019-09-04 20:33:33,154] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56213): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-04 20:33:33,157] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=456492168047, lastReceivedTime=456473469994, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-04 20:33:33,174] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-04 20:33:33,175] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-04 20:33:33,175] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-04 20:33:33,189] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-04 20:33:34,802] - 
  INFO root [2019-09-04 20:33:34,803] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:33:34,804] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:33:34,804] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:33:34,804] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:33:34,835] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56258 )
  INFO root [2019-09-04 20:33:34,835] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56258, SOMAXCONN=5.
  INFO root [2019-09-04 20:33:34,836] - server-configs:
  INFO root [2019-09-04 20:33:34,836] - 	internalTcpAddr : 192.168.56.1:56258
  INFO root [2019-09-04 20:33:34,836] - 	componentID : 6000
  WARN root [2019-09-04 20:33:34,845] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:33:36,359] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 15016 ----
  INFO root [2019-09-04 20:33:36,360] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:33:36,360] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:33:36,360] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:33:36,758] - ---- cellappmgr is running ----
 DEBUG root [2019-09-04 20:33:36,759] - Components::process(): Request for the process of identity...
  INFO root [2019-09-04 20:33:43,883] - Components::findComponents: find logger(1)...
  INFO root [2019-09-04 20:33:43,897] - Components::findComponents: found logger, addr:192.168.56.1:56261
  INFO root [2019-09-04 20:33:43,897] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:33:43,897] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:33:43,899] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-04 20:33:44,003] - Components::findComponents: found dbmgr, addr:192.168.56.1:56263
  INFO root [2019-09-04 20:33:44,003] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:33:44,003] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:33:44,004] - Components::findComponents: found dbmgr, addr:192.168.56.1:56263
  WARN root [2019-09-04 20:33:45,005] - Forwarding logs to logger[192.168.56.1:56261/0/0/0]...
 ERROR root [2019-09-04 20:34:29,778] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=456619566370, lastReceivedTime=456563505610, inactivityExceptionPeriod=134887500)
  INFO root [2019-09-04 20:34:31,538] - 
  INFO root [2019-09-04 20:34:31,550] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:34:31,551] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:34:31,551] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:34:31,551] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:34:31,605] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56311 )
  INFO root [2019-09-04 20:34:31,605] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56311, SOMAXCONN=5.
  INFO root [2019-09-04 20:34:31,605] - server-configs:
  INFO root [2019-09-04 20:34:31,605] - 	internalTcpAddr : 192.168.56.1:56311
  INFO root [2019-09-04 20:34:31,605] - 	componentID : 6000
  WARN root [2019-09-04 20:34:31,614] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:34:33,124] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 2308 ----
  INFO root [2019-09-04 20:34:33,125] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:34:33,125] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:34:33,125] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:34:33,518] - ---- cellappmgr is running ----
 DEBUG root [2019-09-04 20:34:33,520] - Components::process(): Request for the process of identity...
  INFO root [2019-09-04 20:34:40,667] - Components::findComponents: find logger(1)...
  INFO root [2019-09-04 20:34:40,687] - Components::findComponents: found logger, addr:192.168.56.1:56314
  INFO root [2019-09-04 20:34:40,687] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:34:40,687] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:34:40,688] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-04 20:34:40,692] - Components::findComponents: found dbmgr, addr:192.168.56.1:56312
  INFO root [2019-09-04 20:34:40,692] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:34:40,692] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:34:40,794] - Components::findComponents: found dbmgr, addr:192.168.56.1:56312
  WARN root [2019-09-04 20:34:41,795] - Forwarding logs to logger[192.168.56.1:56314/0/0/0]...
 ERROR root [2019-09-04 20:34:56,328] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=456679302694, lastReceivedTime=456644114452, inactivityExceptionPeriod=134887500)
  INFO root [2019-09-04 20:34:56,330] - The following logs sent to logger failed:
  WARN root [2019-09-04 20:34:56,331] - ==> [2019-09-04 20:34:56 324] TCPPacketReceiver::processPendingEvents(192.168.56.1:56314): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-04 20:34:56,331] - ==> [2019-09-04 20:34:56 328] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=456679302694, lastReceivedTime=456644114452, inactivityExceptionPeriod=134887500)
  WARN root [2019-09-04 20:34:56,428] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56312): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-04 20:34:56,429] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=456679530588, lastReceivedTime=456653560495, inactivityExceptionPeriod=134887500)
  INFO root [2019-09-04 20:34:56,440] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-04 20:34:56,443] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-04 20:34:56,443] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-04 20:34:56,444] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-04 20:34:58,097] - 
  INFO root [2019-09-04 20:34:58,098] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:34:58,099] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:34:58,099] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:34:58,100] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:34:58,121] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56353 )
  INFO root [2019-09-04 20:34:58,121] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56353, SOMAXCONN=5.
  INFO root [2019-09-04 20:34:58,121] - server-configs:
  INFO root [2019-09-04 20:34:58,121] - 	internalTcpAddr : 192.168.56.1:56353
  INFO root [2019-09-04 20:34:58,121] - 	componentID : 6000
  WARN root [2019-09-04 20:34:58,122] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:34:59,626] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 12916 ----
  INFO root [2019-09-04 20:34:59,627] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:34:59,627] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:34:59,627] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:35:00,012] - ---- cellappmgr is running ----
 DEBUG root [2019-09-04 20:35:00,013] - Components::process(): Request for the process of identity...
  INFO root [2019-09-04 20:35:07,188] - Components::findComponents: find logger(1)...
  INFO root [2019-09-04 20:35:07,211] - Components::findComponents: found logger, addr:192.168.56.1:56356
  INFO root [2019-09-04 20:35:07,211] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:35:07,211] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:35:07,212] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-04 20:35:07,315] - Components::findComponents: found dbmgr, addr:192.168.56.1:56359
  INFO root [2019-09-04 20:35:07,316] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:35:07,316] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:35:07,316] - Components::findComponents: found dbmgr, addr:192.168.56.1:56359
  WARN root [2019-09-04 20:35:08,317] - Forwarding logs to logger[192.168.56.1:56356/0/0/0]...
 ERROR root [2019-09-04 20:35:50,942] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=456802185221, lastReceivedTime=456750927254, inactivityExceptionPeriod=134887373)
  INFO root [2019-09-04 20:35:50,946] - The following logs sent to logger failed:
  WARN root [2019-09-04 20:35:50,946] - ==> [2019-09-04 20:35:50 941] TCPPacketReceiver::processPendingEvents(192.168.56.1:56356): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-04 20:35:50,947] - ==> [2019-09-04 20:35:50 943] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=456802185221, lastReceivedTime=456750927254, inactivityExceptionPeriod=134887373)
  WARN root [2019-09-04 20:35:51,042] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56359): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-04 20:35:51,046] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=456802420014, lastReceivedTime=456751061918, inactivityExceptionPeriod=134887373)
  INFO root [2019-09-04 20:35:51,064] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-04 20:35:51,064] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-04 20:35:51,064] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-04 20:35:51,065] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-04 20:56:08,629] - 
  INFO root [2019-09-04 20:56:08,636] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:56:08,640] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:56:08,640] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:56:08,641] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:56:08,692] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56638 )
  INFO root [2019-09-04 20:56:08,692] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56638, SOMAXCONN=5.
  INFO root [2019-09-04 20:56:08,692] - server-configs:
  INFO root [2019-09-04 20:56:08,692] - 	internalTcpAddr : 192.168.56.1:56638
  INFO root [2019-09-04 20:56:08,692] - 	componentID : 6000
  WARN root [2019-09-04 20:56:08,693] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:56:10,199] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 16816 ----
  INFO root [2019-09-04 20:56:10,200] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:56:10,200] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:56:10,200] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:56:10,599] - ---- cellappmgr is running ----
 DEBUG root [2019-09-04 20:56:10,611] - Components::process(): Request for the process of identity...
  WARN root [2019-09-04 20:56:14,036] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20792, errorno=10048
  WARN root [2019-09-04 20:56:14,381] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20863, errorno=10048
  INFO root [2019-09-04 20:56:18,422] - Components::findComponents: find logger(1)...
  INFO root [2019-09-04 20:56:18,461] - Components::findComponents: found logger, addr:192.168.56.1:56634
  INFO root [2019-09-04 20:56:18,461] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:56:18,461] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:56:18,463] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-04 20:56:18,467] - Components::findComponents: found logger, addr:192.168.56.1:56634
  INFO root [2019-09-04 20:56:18,471] - Components::findComponents: found dbmgr, addr:192.168.56.1:56632
  INFO root [2019-09-04 20:56:18,471] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:56:18,471] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:56:18,572] - Components::findComponents: found dbmgr, addr:192.168.56.1:56632
  WARN root [2019-09-04 20:56:19,572] - Forwarding logs to logger[192.168.56.1:56634/0/0/0]...
 ERROR root [2019-09-04 20:56:50,181] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=459635476412, lastReceivedTime=459609469826, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-04 20:56:50,187] - The following logs sent to logger failed:
  WARN root [2019-09-04 20:56:50,188] - ==> [2019-09-04 20:56:50 180] TCPPacketReceiver::processPendingEvents(192.168.56.1:56634): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-04 20:56:50,189] - ==> [2019-09-04 20:56:50 182] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=459635476412, lastReceivedTime=459609469826, inactivityExceptionPeriod=134887628)
  WARN root [2019-09-04 20:56:50,288] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56632): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-04 20:56:50,297] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=459635736758, lastReceivedTime=459609320449, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-04 20:56:50,312] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-04 20:56:50,326] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-04 20:56:50,326] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-04 20:56:50,327] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-04 20:57:33,887] - 
  INFO root [2019-09-04 20:57:33,889] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:57:33,890] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:57:33,890] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:57:33,891] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:57:33,910] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56694 )
  INFO root [2019-09-04 20:57:33,910] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56694, SOMAXCONN=5.
  INFO root [2019-09-04 20:57:33,910] - server-configs:
  INFO root [2019-09-04 20:57:33,911] - 	internalTcpAddr : 192.168.56.1:56694
  INFO root [2019-09-04 20:57:33,911] - 	componentID : 6000
  WARN root [2019-09-04 20:57:33,911] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:57:35,416] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 19472 ----
  INFO root [2019-09-04 20:57:35,416] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:57:35,417] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:57:35,417] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:57:35,847] - ---- cellappmgr is running ----
 DEBUG root [2019-09-04 20:57:35,848] - Components::process(): Request for the process of identity...
  INFO root [2019-09-04 20:57:39,421] - 
  INFO root [2019-09-04 20:57:39,429] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:57:39,430] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:57:39,430] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:57:39,430] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:57:39,466] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56708 )
  INFO root [2019-09-04 20:57:39,466] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56708, SOMAXCONN=5.
  INFO root [2019-09-04 20:57:39,466] - server-configs:
  INFO root [2019-09-04 20:57:39,466] - 	internalTcpAddr : 192.168.56.1:56708
  INFO root [2019-09-04 20:57:39,466] - 	componentID : 6000
  WARN root [2019-09-04 20:57:39,467] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-04 20:57:39,810] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20671, errorno=10048
  INFO root [2019-09-04 20:57:39,813] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 15764 ----
  INFO root [2019-09-04 20:57:39,814] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:57:39,814] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:57:39,814] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:57:40,217] - ---- cellappmgr is running ----
 DEBUG root [2019-09-04 20:57:40,219] - Components::process(): Request for the process of identity...
  WARN root [2019-09-04 20:57:43,637] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20453, errorno=10048
  INFO root [2019-09-04 20:57:47,677] - Components::findComponents: find logger(1)...
  INFO root [2019-09-04 20:57:47,743] - Components::findComponents: found logger, addr:192.168.56.1:56705
  INFO root [2019-09-04 20:57:47,744] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:57:47,744] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:57:47,745] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-04 20:57:47,847] - Components::findComponents: found dbmgr, addr:192.168.56.1:56710
  INFO root [2019-09-04 20:57:47,848] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:57:47,848] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:57:47,848] - Components::findComponents: found dbmgr, addr:192.168.56.1:56710
  WARN root [2019-09-04 20:57:48,848] - Forwarding logs to logger[192.168.56.1:56705/0/0/0]...
 ERROR root [2019-09-04 20:58:31,945] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=459864444919, lastReceivedTime=459813778925, inactivityExceptionPeriod=134887500)
  INFO root [2019-09-04 20:58:31,949] - The following logs sent to logger failed:
  WARN root [2019-09-04 20:58:31,949] - ==> [2019-09-04 20:58:31 943] TCPPacketReceiver::processPendingEvents(192.168.56.1:56705): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-04 20:58:31,949] - ==> [2019-09-04 20:58:31 946] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=459864444919, lastReceivedTime=459813778925, inactivityExceptionPeriod=134887500)
  WARN root [2019-09-04 20:58:32,043] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56710): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-04 20:58:32,047] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=459864673930, lastReceivedTime=459813967290, inactivityExceptionPeriod=134887500)
  INFO root [2019-09-04 20:58:32,061] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-04 20:58:32,065] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-04 20:58:32,066] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-04 20:58:32,066] - ThreadPool::destroy(): starting size 2.
